scylla-cql/src/macros.rs

```diff
@@ -42,7 +46,6 @@ pub use scylla_macros::ValueList;
 /// struct MyUdt {
 ///     a: i32,
 ///     b: Option<String>,
-///     c: Vec<u8>,
```

Comment:
Hmm, I assume that you remove this field to show the new behavior, but I wonder whether it won't be confusing - especially for those readers which skim through the description right to the example. Perhaps a short comment in place of the removed field would be in order?